tests/api/v1/foods_api_endpoint.spec.js

```diff
@@ -38,16 +38,53 @@ describe('foods api endpoint', () => {
         updatedAt: new Date()
       }
     ])
-
+    .then(food => {
     return request(app)
     .get('/api/v1/foods')
     .then(response => {
       expect(response.statusCode).toBe(200)
-
       expect(response.body.length).toBe(4)
       expect(Object.keys(response.body[0])).toContain('id')
       expect(Object.keys(response.body[0])).toContain('name')
       expect(Object.keys(response.body[0])).toContain('calories')
     })
+    })
+  });
+  test('user can return a single food', () => {
+    return Food.bulkCreate([
+      {
+        id: 6,
+        name: 'Mango',
+        calories: 120,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        id: 8,
+        name: 'Orange',
+        calories: 80,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      }
+    ])
+    .then(food =>{
+      return request(app)
+      .get('/api/v1/foods/6')
+      .then(response => {
+        expect(response.statusCode).toBe(200)
+        expect(Object.values(response.body)).toContain(6)
+        expect(Object.values(response.body)).toContain('Mango')
+        expect(Object.values(response.body)).toContain(120)
+        expect(Object.keys(response.body)).not.toContain('createdAt')
```

Comment:
Good call on the .not, we're going out of our way to not include those attributes, might as well test for it.